<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Annuaire_view</class>
 <widget class="QMainWindow" name="Annuaire_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TPAnnuaire</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>7</y>
      <width>871</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Recherche</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>887</x>
      <y>6</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>37</y>
      <width>971</width>
      <height>231</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Nom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Prénom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email </string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Téléphone</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Adresse</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ville</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Code</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Groupe</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="message">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>274</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 0);</string>
    </property>
    <property name="text">
     <string>Export réussi ! </string>
    </property>
   </widget>
   <zorder>treeWidget</zorder>
   <zorder>lineEdit</zorder>
   <zorder>pushButton</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>message</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuContact">
    <property name="title">
     <string>Contact</string>
    </property>
    <addaction name="actionNouveau"/>
    <addaction name="actionModifier"/>
    <addaction name="actionExtraire_CSV"/>
   </widget>
   <addaction name="menuContact"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNouveau">
   <property name="text">
    <string>Nouveau</string>
   </property>
  </action>
  <action name="actionSupprimer">
   <property name="text">
    <string>Supprimer</string>
   </property>
  </action>
  <action name="actionExtraire_CSV">
   <property name="text">
    <string>Export CSV</string>
   </property>
  </action>
  <action name="actionNouveau_2">
   <property name="text">
    <string>Nouveau</string>
   </property>
  </action>
  <action name="actionSupprimer_2">
   <property name="text">
    <string>Supprimer</string>
   </property>
  </action>
  <action name="actionSupprimer_3">
   <property name="text">
    <string>Supprimer</string>
   </property>
  </action>
  <action name="actionSupprimer_4">
   <property name="text">
    <string>Supprimer</string>
   </property>
  </action>
  <action name="actionSupprimer_5">
   <property name="text">
    <string>Supprimer</string>
   </property>
  </action>
  <action name="actionImport_CSV">
   <property name="text">
    <string>Import CSV</string>
   </property>
  </action>
  <action name="actionModifier">
   <property name="text">
    <string>Modifier</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
